src/WordOperator.java

```diff
@@ -0,0 +1,15 @@
+public class WordOperator {
```

Comment:
It's better to choose meaningful names for three types of input words, instead of plusWords, minusWords, etc.